plugins/entities/plugin.js

```diff
@@ -80,7 +80,7 @@
 			// final characters.
 			var div = document.createElement( 'div' ),
 				chars;
-			div.innerHTML = '&' + entities.join( ';&' ) + ';';
+			div.innerHTML = CKEDITOR.tools.htmlSafeByReview(('&' + entities.join( ';&' ) + ';'), 'The div element is never added to the page. Also, entities is only set to preset values and used internally');
```

Comment:
Let's investigate where `chars` and `table` goes after here. And what are `entities`?

Reply:
`chars` goes into a new constructed `table` and the table is returned as the output of this function, `buildTable`. `buildTable` is used only twice, both internal and both within the 'entities' plugin's init function which only takes editor as a parameter. The `buildTable` function in both of these cases takes either inline constant entity values or some combination of local const entities and an optional `config.entities_additional`